ansible/setup.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Configure node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    project_user: epoch
+    devops:
+      - dincho
+    developers:
+      - sennui
+      - velzevur
+
+  tasks:
+    - name: Create epoch daemon user
+      user:
+        name: "{{ project_user }}"
+        shell: /bin/bash
+      tags:
+        - project-user
+
+    - name: "Setup root SSH authorized keys"
+      authorized_key:
+        user: "{{ ansible_user }}"
+        key: https://github.com/{{ item }}.keys
+      with_items: "{{ devops }}"
+      tags:
+        - master-ssh-keys
+
+    - name: "Setup {{ project_user }} SSH authorized keys"
+      authorized_key:
+        user: "{{ project_user }}"
+        key: https://github.com/{{ item }}.keys
+      with_items: "{{ developers }} + {{ devops }}"
+      tags:
+        - nodes-ssh-keys
+
+    - name: Add deploy@travis SSH key to authorized keys
+      authorized_key:
+        user: "{{ project_user }}"
+        key: "{{ lookup('file', 'files/travis/deploy_key.pub') }}"
+      tags:
+        - deploy-ssh-keys
+
+    - name: Install dev tools
```

Comment:
I see no `apt update && apt upgrade`. Is that subject to separate periodic procedure?

Reply:
I added automatic apt-get update in 712633aed4166602c389ecdce0de374cff189827

However I prefer to keep upgrades as periodic procedures.